src/appConfig.js

```diff
@@ -1,5 +1,7 @@
 export const COLOR_LIST = ["yellow", "red", "green", "blue", "white"];
 export const USER_TOKEN = "userToken";
+export const INVALID_EMAIL = "Entered value is not email address";
```

Comment:
You can store validation error messages inside of validation.js file

Reply:
moved to validation file